frame/bridge/relayer-game/src/lib.rs

```diff
@@ -160,7 +160,7 @@ decl_storage! {
 		///
 		/// The first key is game id, the second key is round index
 		/// then you will get the affirmations under that round in that game
-		pub Affirmations
+		pub TestCI
```

Comment:
Why make such a change?

Reply:
<img width="1732" alt="image" src="https://user-images.githubusercontent.com/16152305/128466614-320c6de6-9dc5-4258-a3cf-4645d8f662b5.png">
 
In order to test this.